src/renderer.ts

```diff
@@ -9,29 +9,16 @@ import { RuntimeError } from './util';
 // A ContextBuilder is either Renderer.getSVGContext or Renderer.getCanvasContext.
 export type ContextBuilder = typeof Renderer.getSVGContext | typeof Renderer.getCanvasContext;
 
+// eslint-disable-next-line
+function isRenderContext(obj: any): obj is RenderContext {
```

Comment:
Is this our type guard? Seems like a funny way to check if it's a render context, but if it works, it works.

Reply:
It is our type guard unfortunately. By design, there's a big overlap between the interfaces in `RenderContext` and `CanvasRenderingContext2D', so this is the most robust test I can think of.

I think a better fix would be to make `RenderContext` an abstract base class, but that's a fairly big change that I'd like to make in isolation.